data/v2/base/exceptions/error-500.php

```diff
@@ -0,0 +1,17 @@
+<?php
+namespace Elementor\Data\V2\Base\Exceptions;
+
```

Comment:
Add for all files

if ( ! defined( 'ABSPATH' ) ) {
	exit; // Exit if accessed directly
}

Reply:
Handled https://github.com/elementor/elementor/pull/14205/commits/168bd99c71bf7a66472b48dd573266fee062a103